src/components/search/components/searchResults/SearchResults.tsx

```diff
@@ -0,0 +1,223 @@
+/* Import Dependencies */
+import { isEmpty } from 'lodash';
+import { useState } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import { useSearchParams } from 'react-router-dom';
+
+/* Import Utilities */
+import { DetermineTopicDisciplineIcon } from 'app/utilities/NomenclaturalUtilities';
+
+/* Import Config */
+import SearchResultsTableConfig from 'app/config/table/SearchResultsTableConfig';
+
+/* Import Hooks */
+import { useAppSelector, useAppDispatch, useTrigger } from 'app/Hooks';
+
+/* Import Store */
+import { getPhylopicBuild } from 'redux-store/BootSlice';
+import { getSearchDigitalSpecimen, setSearchDigitalSpecimen, getCompareDigitalSpecimen, setCompareDigitalSpecimen } from 'redux-store/SearchSlice';
+
+/* Import Types */
+import { DigitalSpecimen } from 'app/types/DigitalSpecimen';
+import { PaginationObject, Dict } from 'app/Types';
+
+/* Import API */
+import GetPhylopicIcon from 'api/phylopic/GetPhylopicIcon';
+
+/* Import Components */
+import { Paginator } from 'components/elements/Elements';
+import { DataTable, LoadingScreen } from 'components/elements/customUI/CustomUI';
+
+
+/* Data table row type */
+type DataRow = {
+    index: number,
+    DOI: string,
+    taxonomyIconUrl: Promise<string | Dict | undefined>,
+    specimenName: string | undefined,
+    physicalSpecimenID: string | undefined,
+    topicDiscipline: string | undefined,
+    countryOfOrigin: string | undefined,
+    dateCollected: string | undefined,
+    organisation: [string | undefined, string | undefined],
+    selected: boolean
+};
+
+/* Props Type */
+type Props = {
+    pagination: PaginationObject
+};
+
+
+/**
+ * Component that renders the search results table on the search page
+ * @param pagination The pagination object used to index the digital specimen results
+ * @returns JSX Component
+ */
+const SearchResults = (props: Props) => {
+    const { pagination } = props;
+
+    /* Hooks */
+    const dispatch = useAppDispatch();
+    const [searchParams] = useSearchParams();
+    const trigger = useTrigger();
+
+    /* Data table configuration */
+    const { columns } = SearchResultsTableConfig();
+
+    /* Base variables */
+    const searchDigitalSpecimen = useAppSelector(getSearchDigitalSpecimen);
+    const compareDigitalSpecimen = useAppSelector(getCompareDigitalSpecimen);
+    const phylopicBuild = useAppSelector(getPhylopicBuild);
+    const [tableData, setTableData] = useState<DataRow[]>([]);
+    const taxonomyIcons: { [taxonomyIdentification: string]: string | Dict } = {};
+
+    /**
+     * Function to determine the icon being shown with the digital specimen in the search results
+     * @param digitalSpecimen The digital specimen to do an icon check for
+     */
+    const DetermineTableRowIcon = (digitalSpecimen: DigitalSpecimen): Promise<string | Dict | undefined> => {
```

Comment:
Not sure if this should be here

Reply:
Ehh, we could move it out of the component, but it also fills a state variable present here. I could make a hook out of it. Let's see later and discuss.